src/TestHelper/src/System/Diagnostics/RemoteExecutorTestBase.nouap.cs

```diff
@@ -0,0 +1,66 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Diagnostics
+{
+    /// <summary>Base class used for all tests that need to spawn a remote process.</summary>
+    public abstract partial class RemoteExecutorTestBase : FileCleanupTestBase
```

Comment:
Yes I didn't change any code logic. I had to change some internal specifiers to be public specifier instead.